.github/workflows/publish.yml

```diff
@@ -18,13 +18,12 @@ jobs:
       - uses: actions/checkout@v1
 
       - name: Set up Poetry
-        # Need to use version < 1.2.0 in order to support Python 3.6
-        run: pipx install poetry==1.1.15
+        run: pipx install poetry==1.3.2
 
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.7'
+          python-version: '3.9'
```

Comment:
Bit arbitrary - why not 3.11 so that it doesn't need to be updated for as long as possible?

Reply:
Very arbitrary.